components/Trade/Advanced/index.tsx

```diff
@@ -67,18 +79,148 @@ const Advanced: React.FC = styled(({ className }) => {
+        if (positionOverlay) {
+            positionOverlay.removeAttribute('style');
+        }
+
+        // Disable the 'Close Position' button
+        const closeOrderButton = document.getElementById('close-order-button') as HTMLButtonElement;
+        if (closeOrderButton) {
+            closeOrderButton.disabled = true;
+        }
+
+        // Reset Calculator and Margin modal Z-indexes
+        const calculatorEl = document.getElementById('calculator-modal') as HTMLElement;
+        const marginModalEl = document.getElementById('account-modal') as HTMLElement;
+        if (calculatorEl) {
+            calculatorEl.removeAttribute('style');
+        }
+        if (marginModalEl) {
+            marginModalEl.removeAttribute('style');
+        }
+
+        setTutorialComplete();
+    };
+
+    const highlightDots = (e: HTMLDivElement) => {
+        e.addEventListener('click', function () {
+            const navDots: Array<any> = Array.from(
+                document.querySelectorAll('nav[data-tour-elem="navigation"] button'),
+            );
+            // Wait for Reactour to apply styling
+            setTimeout(function () {
+                navDots.map((dot, i) => {
+                    if (dot.classList.contains('reactour__dot--is-active')) {
+                        setCurrentIndex(i);
+                    }
+                });
+                navDots.slice(0, currentIndex).map((dot) => {
+                    dot.classList.add('reactour__dot--is-active');
+                });
+            }, 10);
+        });
+        // Also prevent body scrolling when tour open
+        disableBodyScroll(e);
+    };
+
     return (
-        <div className={`container ${className}`}>
-            <TradingPanel>
-                <MarketSelect account={account ?? ''} />
-                <PlaceOrder selectedTracer={selectedTracer} account={account ?? ''} />
-                <AccountPanel selectedTracer={selectedTracer} account={account ?? ''} order={order} />
-            </TradingPanel>
-            <RightPanel>
-                <TradingView selectedTracer={selectedTracer} />
-            </RightPanel>
-            <Overlay id="trading-overlay" />
-        </div>
+        <>
+            <div className={`container ${className}`}>
+                <TradingPanel>
+                    <MarketSelect account={account ?? ''} />
+                    <PlaceOrder
+                        data-tour-id="placeorder-panel"
+                        selectedTracer={selectedTracer}
+                        account={account ?? ''}
+                    />
+                    <AccountPanel selectedTracer={selectedTracer} account={account ?? ''} order={order} />
+                </TradingPanel>
+                <RightPanel>
+                    <TradingView selectedTracer={selectedTracer} />
+                </RightPanel>
+                <Overlay id="trading-overlay" />
+            </div>
+            {!tourCompleted && (
+                <>
```

Comment:
These are short hand for React.Fragment. When rendering react components it complains if you return multiple children references ie. 
```
return (
    <div />
   <div />
)
```
To avoid this you can wrap it in a react fragment. Which is essentially an invisible div. So it doesnt render an extra DOM element but it keeps react happy. So instead of the above you would do 
```
return (
    <>
         <div />
         <div />
    </>
)
```  

But if you are only returning 1 div then it shouldnt matter. Since <Tour> is the only child youd be able to get away without the React.Fragment in this case